src/interfaces/IPoolManager.sol

```diff
@@ -6,12 +6,11 @@ import {PoolKey} from "../types/PoolKey.sol";
 import {Pool} from "../libraries/Pool.sol";
 import {IHooks} from "./IHooks.sol";
 import {IFees} from "./IFees.sol";
-import {IClaims} from "./IClaims.sol";
 import {BalanceDelta} from "../types/BalanceDelta.sol";
 import {PoolId} from "../types/PoolId.sol";
 import {Position} from "../libraries/Position.sol";
 
-interface IPoolManager is IFees, IClaims {
+interface IPoolManager is IFees {
```

Comment:
Hmm yeah so it's a bit weird because the solmate implementation doesn't use the interface either, so getting a ton of compile errors due to conflicting implementations

Reply:
Resolved offline - going to copy in the solmate implementation locally and inherit the interface